dds-test/CMakeLists.txt

```diff
@@ -25,6 +25,8 @@ find_package(soss-mock REQUIRED)
 find_package(fastrtps REQUIRED)
 
 if(WIN32)
+    find_file(MOCKDLL NAMES "soss-mock.dll" PATHS "${soss-mock_DIR}" PATH_SUFFIXES "lib" ) #pending to check
```

Comment:
`find_package(soss_mock)` doesn't solve this automatically?

Reply:
No, it solves the .a lib, but not the .dll. Windows stuffs... :man_facepalming:

Reply:
The problem is that windows only search a DLL file in the path or in the same executable location.

Reply:
As we have talked, remove the whole section and we will address the Windows issue in the future, modifying the `%PATH%` from the `install/setup.bat` script.

Reply:
Ok, I remove it